cumulusci/utils.py

```diff
@@ -146,6 +144,17 @@ def download_extract_zip(url, target=None, subfolder=None, headers=None):
     return zip_file
 
 
+def download_extract_github(github_repo, subfolder, ref=None):
+    if not ref:
+        ref = github_repo.default_branch
+    zip_content = io.BytesIO()
+    github_repo.archive("zipball", zip_content, ref=ref)
+    zip_file = zipfile.ZipFile(zip_content)
+    subfolder_dir = zip_file.namelist()[0] + subfolder
```

Comment:
I don't think the `zipfile` module guarantees any particular ordering of the namelist, so it would probably be safest to sort it first.